deepmd/pt/model/descriptor/sezm_nn/so2.py

```
@@ -0,0 +1,1625 @@
+                    raise RuntimeError(
+                        "Unexpected SO2Linear output width in bias correction"
+                    )
+                bias_correction = bias0 * (
+                    radial_factor * edge_cache.edge_env.reshape(-1, 1, 1) - 1.0
+                )
+                x_local[:, :, 0, :].add_(bias_correction)
+
+            if self.use_so2_attn_res:
+                so2_depth_sources = [x_local]
+                for layer_idx, (so2_linear, inter_norm, non_linear) in enumerate(
+                    zip(self.so2_linears, self.so2_inter_norms, self.non_linearities)
```

Ruff (0.15.13)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)